### ISSUE TITLE
html_safe values lose safety

### ISSUE THREAD

**Issue by glebm** [svenfuchs/i18n#377]
Given `config/locales/test.en.rb`:

```ruby
{
    en: {
        test: '<i>test</i>'.html_safe,
    }
}
```

I expect that:

```ruby
I18n.t('test').html_safe? #=> true
```

In v0.8.5 this actually returns `false`.
This used to work but I'm not sure exactly at which version this stopped working.

**Comment by radar** [comment]
Hey @glebm :wave:

Thanks for reporting this issue. I attempted to reproduce this issue today and was unable to. Here are my exact steps:

1. Create a new rails app `rails new i18n-377`
2. Create `config/locales/test.en.rb` in this Rails app with the content specified
3. In the Gemfile, specify `gem 'i18n', '0.8.5'` explicitly.
4. Create a new file called `repro.rb` at the rails app root, with content of `p I18n.t('test').html_safe? `
5. Run this file with `rails runner repro.rb`
6. Observe the output is `true`, not `false`.

Could you please try to reproduce this in a new app and if you're able to do so, put that up on GitHub? 

Thanks!

**Comment by glebm** [comment]
Hi @radar!

It must be one of the gems I have causing this.
I'll need to add them to a demo app one by one to figure out which one is causing this.

This might take a while :sweat_smile: Closing the issue for the time being

**Comment by glebm** [comment]
A surprising development! The issue is fixed in v0.8.6. :smile:

**Comment by radar** [comment]
Hooray! Thanks for letting me know :)
